packages/grafana-ui/src/components/DateTimePickers/TimeRangePicker.tsx

```diff
@@ -2,7 +2,8 @@ import { css, cx } from '@emotion/css';
 import { useDialog } from '@react-aria/dialog';
 import { FocusScope } from '@react-aria/focus';
 import { useOverlay } from '@react-aria/overlays';
-import { memo, createRef, useState, useEffect } from 'react';
+import { memo, createRef, useState, useEffect, useCallback } from 'react';
+import usePrevious from 'react-use/lib/usePrevious';
```

Comment:
```suggestion
import { usePrevious } from 'react-use';
```

Reply:
Isn't the `'react-use/lib/usePrevious'` form better for tree shaking?

Reply:
I think our bundlers should be able to optimise this and this is the syntax we use everywhere else in the code.